tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md

```diff
@@ -528,6 +528,11 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIUpload/upload.feature:138](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIUpload/upload.feature#L138)
 -   [webUIUpload/uploadEdgecases.feature:67](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIUpload/uploadEdgecases.feature#L67)
 
+### [user cannot upload new folder/file into folder Shares](https://github.com/owncloud/ocis/issues/2322)
```

Comment:
https://github.com/owncloud/ocis/issues/2322 https://github.com/cs3org/reva/pull/1939 should have fixed this?

And I think that code is in oCIS now. But maybe some web code needs to also be adjusted to get these passing.

Anyway, we had better rebase this PR to make sure that we get an up-to-date CI result before merging.